data/json/recipes/food/meat_dishes.json

```diff
@@ -23,5 +23,130 @@
       [ [ "meat_red_raw", 1, "LIST" ], [ "fish", 1 ], [ "meat_offal", 1, "LIST" ] ],
       [ [ "salt_preservation", 1, "LIST" ] ]
     ]
+  },
+  {
+    "type": "recipe",
+    "result": "kiymali_pide",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_MEAT",
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "time": "2 hours",
+    "batch_time_factors": [ 80, 1 ],
+    "book_learn": [ [ "cookbook_turkish", 2 ] ],
+    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "COOK", "level": 2 } ],
+    "tools": [ [ [ "surface_heat", 30, "LIST" ] ] ],
+    "components": [
+      [ [ "flatbread", 1 ] ],
+      [ [ "meat_red_raw", 1, "LIST" ] ],
+      [ [ "salt", 1 ] ],
+      [ [ "onion", 1 ] ],
+      [ [ "tomato", 2 ] ],
+      [ [ "any_butter_or_oil", 1, "LIST" ] ],
+      [ [ "sugar", 1 ] ],
+      [ [ "bell_pepper", 1 ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "result": "lahmacun",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_MEAT",
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "time": "30 minutes",
+    "batch_time_factors": [ 80, 1 ],
+    "book_learn": [ [ "cookbook_turkish", 2 ] ],
+    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "COOK", "level": 2 } ],
+    "tools": [ [ [ "surface_heat", 40, "LIST" ] ] ],
+    "components": [
+      [ [ "flatbread", 1 ] ],
+      [ [ "meat_red_raw", 1, "LIST" ] ],
+      [ [ "salt", 1 ] ],
+      [ [ "onion", 1 ] ],
+      [ [ "tomato", 1 ] ],
+      [ [ "any_butter_or_oil", 1, "LIST" ] ],
+      [ [ "bell_pepper", 1 ] ],
+      [ [ "garlic_clove", 2 ] ],
+      [ [ "seasoning_mild", 2, "LIST" ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "result": "iskender",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_MEAT",
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "time": "90 minutes",
+    "batch_time_factors": [ 80, 1 ],
+    "book_learn": [ [ "cookbook_turkish", 2 ] ],
+    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "COOK", "level": 2 } ],
+    "tools": [ [ [ "surface_heat", 40, "LIST" ] ] ],
+    "components": [
+      [ [ "meat_red_raw", 1, "LIST" ] ],
+      [ [ "salt", 1 ] ],
+      [ [ "pepper", 1 ] ],
+      [ [ "onion", 2 ] ],
+      [ [ "tomato", 4 ] ],
+      [ [ "bell_pepper", 2 ] ],
+      [ [ "garlic_clove", 2 ] ],
+      [ [ "seasoning_mild", 2, "LIST" ] ],
+      [ [ "yoghurt", 1 ] ]
```

Comment:
Not necessarily your problem to fix but if I'm reading the json correctly the only way to make yoghurt is to already have some yoghurt in game currently.   Someone should probably fix that.